### PR TITLE
Implement read for int64

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**

### CODE REVIEW

**Comment by Thirumalai-Shaktivel** [comment]
@Smit-create Do you want to make any changes?

File: integration_tests/file_01_data.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 10130
 c
 4.20
+4024052
```
Comment (Shaikh-Ubaid) [inline integration_tests/file_01_data.txt:4]: I would use a value that does not fit in a 32-bit integer but fits in a 64-bit integer (in the orders of 10^15). I think that would make it a more robust test.

File: src/libasr/codegen/asr_to_llvm.cpp [hunk 2]
```diff
@@ -6509,20 +6509,30 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
...
+                    fn = module->getFunction(runtime_func_name);
+                    if (!fn) {
+                        llvm::FunctionType *function_type = llvm::FunctionType::get(
+                                llvm::Type::getVoidTy(context), {
+                                    llvm::Type::getInt64Ty(context)->getPointerTo(),
+                                    llvm::Type::getInt32Ty(context)
+                                }, false);
+                        fn = llvm::Function::Create(function_type,
+                                llvm::Function::ExternalLinkage, runtime_func_name, *module);
+                    }
```
Comment (Shaikh-Ubaid) [inline src/libasr/codegen/asr_to_llvm.cpp:6535]: We can use `std::string runtime_func_name` and `llvm::Type* type_arg` and have single code for function/function-type  creation. Please see the real case below for example.

**Review by certik — APPROVED** [review]
Let's use a larger integer as @Shaikh-Ubaid said, then I think we can merge it, the rest we can do later.

**Review by Smit-create — APPROVED** [review]

File: integration_tests/file_01_data.txt (hunk 1 above)
Comment (Thirumalai-Shaktivel) [reply integration_tests/file_01_data.txt]: Done

File: src/libasr/codegen/asr_to_llvm.cpp (hunk 2 above)
Comment (Thirumalai-Shaktivel) [reply src/libasr/codegen/asr_to_llvm.cpp]: Done

File: integration_tests/file_01.f90 [hunk 3]
```diff
@@ -2,18 +2,21 @@ program fileops
...
+    read(j, *) d
     rewind(j)
     close(j)
 
-    print *, i, c, z
+    print *, i, c, z, d
     if (i /= 10130) error stop
     if (c /= "c") error stop
     if (abs(z - 4.20) > 1e-6) error stop
+    if (d /= 9223372036854775804) error stop
```
Comment (Shaikh-Ubaid) [inline integration_tests/file_01.f90:20]: I think this should be
```suggestion
    if (d /= 9223372036854775804_8) error stop
```

**Review by Shaikh-Ubaid — APPROVED** [review]
It looks good to me. Thank you so much for fixing this!

**Pull request merged**